.github/workflows/trivy.yaml

```
@@ -0,0 +1,105 @@
+    if: |
+      github.event.workflow_run.conclusion == 'success' &&
+      github.event.workflow_run.event == 'release'
+    steps:
+      - name: Scan frontend image
+        uses: aquasecurity/trivy-action@6c175e9c4083a92bbca2f9724c8a5e33bc2d97a5
+        with:
+          image-ref: ${{ env.DOCKERHUB_USERNAME }}/owasp-nest-frontend:production
+          severity: 'HIGH,CRITICAL'
+          ignore-unfixed: true
+          ignore-cves:
+            - CVE-2024-56171
```

actionlint (1.7.4)

expected scalar node for string value but found sequence node with "!!seq" tag

(syntax-check)